tests/test_sticker.py

```diff
@@ -325,6 +325,24 @@ async def test_error_without_required_args(self, bot, chat_id):
         with pytest.raises(TypeError):
             await bot.send_sticker(chat_id)
 
+    @flaky(3, 1)
+    async def test_premium_animation(self, bot):
+        # testing animation sucks a bit since we can't create a premium sticker. What we can do is
+        # get a sticker set which includes a premium sticker and check that specific one.
+        premium_sticker_chat = await bot.get_sticker_set("Flame")
+        # the first one to appear here is a sticker with unique file id of AQADOBwAAifPOElr
+        # this could change in the future ofc.
+        premium_sticker = premium_sticker_chat.stickers[20]
+        assert premium_sticker.premium_animation.file_unique_id == "AQADOBwAAifPOElr"
```

Comment:
is it of any value to us to assert the specific unique file id here? wouldn't it suffice to check that it's a non-empty string? that way the tests won't fail if the sticker set changes

Reply:
Its one improvement to make sure we set the string to the proper value imo. If we mess up in the implementation and set file_unique_id = "a", your test wouldn't catch it.

I am very much fine with changing this as soon as we see that the test fail because of telegram changing the set, but I doubt it will happen